third_party/blink/renderer/core/brave_page_graph/CHANGES.md

```diff
@@ -0,0 +1,7 @@
+PageGraph Version Changes
+===
+
+0.4
+---
+ - Add "frame id" properties for most edges, to resolve ambiguities in the
```

Comment:
@goodov do you mean rename the file to changelog.md? Or move the contents of this file to a more central changelog.md?

I'd prefer the former, since the changes I plan on adding here will just be things relevant to graph generation and structure, and so will only be of interest to other folks using pagegraph (and not useful at all to general Brave users)

Reply:
I've rebased and renamed to changelog.md